app/prod/requirements.txt

```diff
@@ -0,0 +1,14 @@
+black~=22.3.0
```

Comment:
I would remove black, drf-spectacular, pydot, and maybe django-linear-migrations from the production build. I don't think there's any use for them outside development.